src/control/cmd/dmg/pretty/printers_test.go

```diff
@@ -133,18 +142,69 @@ Hosts Command  Error
 ----- -------  -----             
 host1 hostname command not found 
 `,
+			expErr: nil,
+		},
+		"Two host, same error": {
+			cmd: "hostname",
+			resp: &control.CollectLogResp{
+				HostErrorsResp: control.MockHostErrorsResp(t,
+					&control.MockHostError{
+						Hosts: "host1",
+						Error: "command not found",
+					},
+					&control.MockHostError{
+						Hosts: "host2",
+						Error: "command not found",
+					}),
+			},
+			expStdout: `
+Hosts     Command  Error             
+-----     -------  -----             
+host[1-2] hostname command not found 
+`,
+			expErr: nil,
+		},
+		"Two host, different error": {
+			cmd: "hostname",
+			resp: &control.CollectLogResp{
+				HostErrorsResp: control.MockHostErrorsResp(t,
+					&control.MockHostError{
+						Hosts: "host1",
+						Error: "command not found",
+					},
+					&control.MockHostError{
+						Hosts: "host2",
+						Error: "command not available",
+					}),
+			},
+			expStdout: `
+Hosts Command  Error                 
+----- -------  -----                 
+host1 hostname command not found     
+host2 hostname command not available 
+`,
+			expErr: nil,
 		},
 	} {
 		t.Run(name, func(t *testing.T) {
 			var out strings.Builder
+			var err error
 
-			if err := UpdateErrorSummary(tc.resp, tc.cmd, &out); err != nil {
-				t.Fatal(err)
+			if tc.resp == nil {
+				err = UpdateErrorSummary(nil, tc.cmd, &out)
```

Comment:
if you are checking for a "nil interface" inside the function then you shouldn't need this hack

Reply:
This was older code. It has been changed in later patch. Double check and let me know if that needs to be changed,

Reply:
yes I saw the update, I was going through the individual commits and saw you updated later, nice work!